source/deployment/sso-saml-before-you-begin.rst

```diff
@@ -1,7 +1,13 @@
-Before you begin
+Before You Begin
 ----------------
 
-Before you configure SAML:
+Starting with version 5.20, a new SAML library implementation is available. This implementation does not require the installation of the XML Security Library. It is recommended that you use this new implementation as the old implementation will eventually be deprecated.
+
+To use the new implementation, navigate to **System Console > Experimental Features > Use Improved SAML Library**. Activate the feature and choose **Save** and then restart the server. The configuration change will not take effect until the server is restarted.
+
+To change the configuration file directly, edit the “UseNewSamlLibrary” setting under “ExperimentalSettings”. 
```

Comment:
Change it in the config.json file?

Reply:
Yes.  The config.json file. And the setting is actually "UseNewSAMLLibrary". I probably had it wrong in my documentation. 

"ExperimentalSettings": {
        "ClientSideCertEnable": false,
        "ClientSideCertCheck": "secondary",
        "EnableClickToReply": false,
        "LinkMetadataTimeoutMilliseconds": 5000,
        "RestrictSystemAdmin": false,
        "UseNewSAMLLibrary": true
    },